scraper/keldoc/keldoc_center.py

```diff
@@ -39,10 +38,14 @@ def fetch_vaccine_cabinets(self):
             cabinet_url = API_KELDOC_CABINETS.format(self.id, specialty)
             try:
                 cabinet_req = self.client.get(cabinet_url)
-            except TimeoutException:
+                cabinet_req.raise_for_status()
+            except httpx.TimeoutException as hex:
                 logger.warning(f"Keldoc request timed out for center: {self.base_url} (vaccine cabinets)")
                 continue
-            cabinet_req.raise_for_status()
+            except httpx.HTTPStatusError as hex:
+                logger.warning(f"Keldoc request returned error {hex.response.status_code} "
+                               f"for center: {self.base_url} (vaccine cabinets)")
+                return False
```

Comment:
Attention en cas d'exception il faut passer à la suite du for (continue) comme pour l'exception Timeout et ne pas simplement sortir de la fonction

Reply:
J'ai uniformisé tout ça, bien vu, je devais être fatiguée.
Je me demandais, il y a une commande pour envoyer une recherche sur seulement un centre ?